src/Microsoft.VisualStudio.LanguageServices.Razor/VisualStudioRazorSyntaxFactsService.cs

```diff
@@ -1,15 +1,13 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
+using System.Composition;
 using Microsoft.CodeAnalysis.Razor;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
-    public interface ITagHelperResolver
+    [Export(typeof(RazorSyntaxFactsService))]
```

Comment:
It would be nice to have a big scary banner here saying that this is legacy and should be transitioned away from (on all of these)